### ISSUE TITLE
Search for salinity in the last day only returns 3 datasets

### ISSUE THREAD

**Issue by rsignell-usgs** [rsignell-usgs/ooici-erddap#16]
This search for `sea_water_practical_salinity` in the last day:  

https://erddap-uncabled.oceanobservatories.org/uncabled/erddap/search/advanced.html?page=1&itemsPerPage=1000&searchFor=&protocol=%28ANY%29&cdm_data_type=%28ANY%29&institution=%28ANY%29&ioos_category=%28ANY%29&keywords=%28ANY%29&long_name=%28ANY%29&standard_name=sea_water_practical_salinity&variableName=%28ANY%29&maxLat=&minLon=&maxLon=&minLat=&minTime=2017-12-28

only returns 3 datatsets (one of them is incorrect, as raised in #15), but there should be at least 10 datasets with salinity from Pioneer alone, right @spetillo?

**Comment by spetillo** [comment]
@rsignell-usgs I dug into this come more... Searching back in time we see a lot more data sets:
https://erddap-uncabled.oceanobservatories.org/uncabled/erddap/search/index.html?page=1&itemsPerPage=1000&searchFor=sea_water_practical_salinity

Regardless, there are still missing Pioneer CTD salinity data sets even from that superset:
Each of the 7 deployed Pioneer Profiler Moorings (cp01cnpm, cp02pmci, cp02pmco, cp02pmui, cp02pmuo, cp03ispm, cp04ospm) should have `sea_water_practical_salinity` calculated from the Wire Following Profiler's CTD (I only see these data from cp02pmci & cp02pmco in the link above).

I have verified that I have found the following Pioneer & Global CTD salinity data sets in **older** data: 
The Pioneer Surface Moorings (cp01cnsm, cp03issm, cp04ossm) should each have two CTDBPs (one on the NSIF and one on the MFN) that produce a conductivity from which you can calculate  `sea_water_practical_salinity` (I see all of these on ERDDAP). 
Similarly, each Global Surface Mooring (ga01sumo, gi01sumo, and gs01sumo) each have one CTDBP on the NSIF (I also see all of these, plus the Riser CTDBPs & CTDMOs).

There are also a number of other instruments whose data streams contain sea_water_practical_salinity that I have not done an accounting of here, but many appear in the search link above.

**Summary**: 
This actually brings us to 16+ data sets (when all moorings are functional) that should have CTD salinity values calculated for them and up on ERDDAP from the WFP CTDs and non-Riser CTDBPs alone. Of this, I see 5 Pioneer Profiler Moorings' WFP CTD data sets missing going back as far as there is currently data in ERDDAP. For the last 2 days that @rsignell-usgs was searching on, there are only 2 available of these 14+ data sets that should be on ERDDAP right now (gi01sumo and gs01sumo are not transmitting).

**Comment by rsignell-usgs** [comment]
@spetillo, I don't think there is a problem with ERDDAP advanced search: my advanced search (yeilding 3 datasets) was with for salinity data within the last 2 days:

https://erddap-uncabled.oceanobservatories.org/uncabled/erddap/search/advanced.html?page=1&itemsPerPage=1000&searchFor=sea_water_practical_salinity&protocol=%28ANY%29&cdm_data_type=%28ANY%29&institution=%28ANY%29&ioos_category=%28ANY%29&keywords=%28ANY%29&long_name=%28ANY%29&standard_name=%28ANY%29&variableName=%28ANY%29&maxLat=&minLon=&maxLon=&minLat=&minTime=2017-12-27

might want to just edit your comment above.

**Comment by spetillo** [comment]
@rsignell-usgs Indeed. I missed the date restriction and updated my previous comment to be more accurate. It looks like there are only these 3 data sets (2 good, one bad) going back through 11/15/17. That is 3 days after the last Pioneer deployment completed (maybe the new mooring deployments were not fully configured in datasets.xml?). The ERDDAP is currently missing Salinity in 14+ total WFP CTD and non-Riser CTDBP data sets.

**Comment by rsignell-usgs** [comment]
After months of being down, ERDDAP is back up!   

And a search for `sea_water_practical_salinity` data in the last two days yields 4 datasets:
https://erddap-uncabled.oceanobservatories.org/uncabled/erddap/search/advanced.html?page=1&itemsPerPage=1000&searchFor=&protocol=%28ANY%29&cdm_data_type=%28ANY%29&institution=%28ANY%29&ioos_category=%28ANY%29&keywords=%28ANY%29&long_name=%28ANY%29&standard_name=sea_water_practical_salinity&variableName=%28ANY%29&maxLat=&minLon=&maxLon=&minLat=&minTime=now-2days&maxTime=

@spetillo or @cwingard is 4 datasets correct, or should there be more? 

![2018-12-07_13-56-39](https://user-images.githubusercontent.com/1872600/49667182-51094d00-fa28-11e8-9615-8791b4d62880.png)

**Comment by spetillo** [comment]
@rsignell-usgs There should definitely be more. I see no Pioneer, Global, or Glider CTD representation on this list.
